packages/framework/global/src/utils/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './assert.js';
+export * from './createZodUnion.js';
```

Comment:
Thanks for the enhancement! Could you move this file to `packages/blocks/_common/utils/zod.ts`? There seems to be a better fit than the `global` package.

Reply:
done!